### DISCUSSION TITLE
hub/manual/InstallHub

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#604] [Notes]
# hub/manual/InstallHub

https://docs.unity3d.com/hub/manual/InstallHub.html

**Comment by rhys-vdw** [comment]
Packages for macOS and Windows are also available via Brew and Chocolatey respectively:

https://formulae.brew.sh/cask/unity-hub
https://community.chocolatey.org/packages/unity-hub
